### ISSUE TITLE
(DS2 SotFS) Param Studio remove some data from regulation file after loading in new project

### ISSUE THREAD

**Issue by elaeis** [Philiquaz/DSParamStudio#12]
I try to change the finished mod (Seeker of Fire (https://www.nexusmods.com/darksouls2/mods/978)) for myself and after I upload the data to the program and save it without making any changes, the file size changes by ~8kb (from 884 KB (905 216 bytes) to 876 KB (897 024 bytes)). In game it's change some enemies, for example No-Man's Wharf (it change Gyrm Warrior with 2 Axes to Hammersmith (or Undead Steelworker)), what's bad but i can live with it, but in some places it's break the game, immediately closing it (on Flexile Sentry, after I boarded the ship)

It's bug or i do something wrong? or some sources needed for correct working?

P.S. I (somehow, i can't remember how -_-) edit data and it's not cause this bug (i edit param only) with DSMapStudio involved when editing. If needed i can upload full data from mod ("Seeker*" with patches), my mass edit txt data (348552 rows edited in 21 row using mass edit function) and working files, with some changes (i make backup before changing params in working my version of mod)

P.S.2: I wrote a message through a translator, I could not quite choose the right words to convey the meaning

**Comment by Philiquaz** [comment]
That's interesting that it would change significantly without edits.
As ds2 enemy data is found in params, can you confirm if such a change in enemy type occurs when loading the params or saving them? (aka, which does it appear as inside the program - if indeed it does appear)
Additionally, can you try unpacking the params to make them loose, then load and save those loose params to check if any are not changed/saved by the program?
Sending your version shouldn't be wholly necessary so long as I can reproduce the issue with the instructions you've provided

**Comment by elaeis** [comment]
In Seeker mod some files is loose, and some in regulation (clear install have 7 folders and enc_regulation with 882Kb size in mod folder), maybe this affect (in mod changed both map and enemies\items params)

Loading with loose params not help, still have a problem (some enemies changed and game crash on ship)

I install mod, run game - all ok, load mod files in project folder, open it with studio, save, move files to mod folder in game (delete old files from mod in game folder before moving), run game, catch problem

If i try choose "generator*" param in left menu, param studio crash

I try load project in map studio - without loose same problem 
but if i load with loose and save all - problem not appear...
hmmm... if i load project in map studio, save all, load project in param studio and make changes (both with loose param option), it's worked and problem don't appear..

**Comment by elaeis** [comment]
can this be fixed (there is no need to close the issue), or is the problem how the author of the mod approached its creation?

**Comment by Philiquaz** [comment]
The project folder should be the mod folder, for a start. There should be no need for manually moving files. Perhaps you are at some level re-introducing vanilla params?
Again, confusion between packed and loose params may also introduce unwanted params

**Comment by elaeis** [comment]
No, i just change items and npc params (change damage multiplier for spells, change souls for enemies, change some params in reinforcement and other like this). Problem is this - clear downloaded mod use and loose param (7 game folders - sounds, map, menu, Param and other) and packed param - enc_regulation.bnd.dcx with size 884 KB. If i try change params with param studio only, mod corrupted, but if i use map studio at first, i can change params with param studio fine, just using loose param in both programms. I manually moving files because of testing mod and having backup, so i don't think that this can affect. (i can change params  in ./Game/mod_folder, but mod breaks often... before i find way with map studio and loose files only
